layouts/_default/events.yaml

```diff
@@ -0,0 +1,18 @@
+{{- $events := .Site.GetPage "events" -}}
+---
+events:
+{{- range sort $events.Pages ".Date" "asc" -}}
+{{- $images := .Resources.ByType "image" }}
+{{- $feature := $images.GetMatch .Params.feature }}
+- title: {{ .Params.title }}
+  date: {{ .Date.Format "2006-01-02T15:04:05Z07:00" }}
+  location: {{ .Params.location }}
```

Comment:
This should be only "Online" or "Amsterdam" as values, because global website uses this to determine sorting of multiple time zones (if there are some) and whether to 12 or 24 hour time zones

Reply:
This one might be a bit tricky. This field is free text for us, since we might put the location of a festival etc. We also had the Berlin Tech Workers conference in there: https://techwerkers.nl/en/events/2025-04-11/

we could just hardcode it as "The Netherlands" (we're not just Amsterdam :wink:). What works for y'all?

Reply:
the API expects a key called `locations:` (plural) and for now doesn't do anything with it yet anyways. In future idea would be to prominently display one or more locations, e.g if online and in person; but the free text of actual location also works, given that we know all events are from Amsterdam chapter here anyways. Can add Chapter key in the future

Reply:
I've updated the key to `locations`. Have left out the `chapter` key until it is needed :smile: 

All good?